db_revisions/versions/102fb94a24cc_create_user_action_and_drop_signature_submitter_accepter_tables.py

```diff
@@ -0,0 +1,104 @@
+"""create user_action table to replace Signatures, Submitter, and Accepter tables
+
+Revision ID: 102fb94a24cc
+Revises: ccc50ec18a7e
+Create Date: 2024-04-12 13:33:20.053959
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "102fb94a24cc"
+down_revision: Union[str, None] = "0040045eae14"
```

Comment:
Down revision should now be 4cd30d188352 after the merging of the snapshot PR.  Make sure to update the comment, too ;)